README.md

```diff
@@ -403,6 +403,35 @@ db.on('put', function (key, value) {
 See the [`CONTRIBUTING.md`](https://github.com/Level/community/blob/master/CONTRIBUTING.md) file for more details.
 
 <a name="license"></a>
+## Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
+<a href="graphs/contributors"><img src="https://opencollective.com/level/contributors.svg?width=890" /></a>
```

Comment:
This `src` seems broken?

Reply:
Yes, this is a known issue on open collective, but it is being addressed. I can track and update you here when it is fixed. Should pull from github for all repo contributors.

Reply:
Script fix is in place and is supposed to run at 0000 EST - fingerscrossed